package.json

```diff
@@ -39,7 +39,7 @@
     "xmlhttprequest"
   ],
   "engines": {
-    "node": "^14.19.1 || ^16.14.2 || ^18.0.0 || ^20.0.0"
+    "node": "^14.0.0 || ^16.0.0 || ^18.0.0 || ^20.0.0 || ^21.0.0"
```

Comment:
Honestly would recommend dropping Node 14 and 16 as they're both EOL. Should also probably start testing Node 22 too?

Reply:
Yes, I'm already working on it

Reply:
DONE 🙂🥳